tests/ssl-opt.sh

```diff
@@ -2818,15 +3001,15 @@ run_test    "Encrypt then MAC, DTLS: disabled, empty application data record" \
 ## The ClientHello content is spelled out below as a hex string as
 ## "prefix ciphersuite1 ciphersuite2 ciphersuite3 ciphersuite4 suffix".
 ## The expected response is an inappropriate_fallback alert.
-requires_openssl_with_fallback_scsv
+requires_config_enabled MBEDTLS_SSL_FALLBACK_SCSV
 run_test    "Fallback SCSV: beginning of list" \
```

Comment:
Fixing the dependency of this test case is causing a CI failure on FreeBSD. The failure isn't happening on the base branch because of the incorrect dependency. https://github.com/Mbed-TLS/mbedtls/issues/5731